torch/csrc/jit/script/sugared_value.h

```diff
@@ -157,12 +167,48 @@ struct TORCH_API BuiltinModule : public SugaredValue {
   c10::optional<int64_t> version;
 };
 
+struct TORCH_API UserTypeValue : public SugaredValue {
```

Comment:
Right now this is doing double duty as a type and in instance of that type. I plan to separate those concepts